src/rqt_py_console/py_console_widget.py

```diff
@@ -42,8 +42,11 @@ class PyConsoleWidget(QWidget):
 
     def __init__(self, context=None):
         super(PyConsoleWidget, self).__init__()
-        rp = rospkg.RosPack()
-        ui_file = os.path.join(rp.get_path('rqt_py_console'), 'resource', 'py_console_widget.ui')
+
+        _, package_path = get_resource('packages', 'rqt_py_console')
+        ui_file = os.path.join(
+            package_path, 'share', 'rqt_py_console', 'resource', 'py_console_widget.ui')
```

Comment:
This line needs to be fixed to address #4. It needs a directory named after the package between `resource` and the UI file.